Libraries/Text/RCTShadowText.m

```diff
@@ -415,20 +415,26 @@ - (RCTSizeComparison)attemptScale:(CGFloat)scale
 
   NSRange glyphRange = NSMakeRange(0, textStorage.length);
   [textStorage beginEditing];
-  [textStorage enumerateAttribute:NSFontAttributeName
-                           inRange:glyphRange
-                           options:0
-                        usingBlock:^(UIFont *font, NSRange range, BOOL *stop)
-   {
-     if (font) {
-       UIFont *originalFont = [self.attributedString attribute:NSFontAttributeName
-                                                       atIndex:range.location
-                                                effectiveRange:&range];
-       UIFont *newFont = [font fontWithSize:originalFont.pointSize * scale];
-       [textStorage removeAttribute:NSFontAttributeName range:range];
-       [textStorage addAttribute:NSFontAttributeName value:newFont range:range];
-     }
-   }];
+
+  @try {
+    [textStorage enumerateAttribute:NSFontAttributeName
+                            inRange:glyphRange
+                            options:0
+                         usingBlock:^(UIFont *font, NSRange range, BOOL *stop)
+     {
+       if (font) {
+         UIFont *originalFont = [self.attributedString attribute:NSFontAttributeName
+                                                         atIndex:range.location
```

Comment:
I don't think so, if I remember correctly one is post-scaling, and one is pre-scaling.